src/main/resources/skillresponse/response_en.properties

```diff
@@ -0,0 +1,129 @@
+#
```

Comment:
Just curious - why is there both a `response` and a `response_en` property files?

Reply:
Currently, response and response_en are exactly the same,
I added this because it may regionalized in the future per discussion offline (e.g. EU)
https://docs.oracle.com/javase/7/docs/api/java/util/ResourceBundle.html

in that case, we can just add `response_es` `response_de` if necessary

Reply:
Regionalization is great, but it seems strange to me to have duplicate content. Shouldn't it recognize (or we configure) that the default `response.properties` is English, rather than bundling a duplicate `response_en.properties`? Otherwise we're going to have to maintain two duplicate files, and for future maintainers it may not be obvious. I'm thinking of Android as an example - we have a `strings.xml` with a default language, but if the default language is English, we don't have `strings_en.xml`.